actions/submit-signing-request/helper-artifact-download.ts

```diff
@@ -68,11 +68,14 @@ export class HelperArtifactDownload {
     }
 
     private resolveOrCreateDirectory(relativePath:string): string {
```

Comment:
Rename parameter from `relativePath` to `path`, since it can be both absolute and relative now